docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,154 @@
+tags:
+  - Findability
+  - Galaxy
+  - Community specific
+  - Tools
+  - EDAM
+  - bio.tools
+  - Metadata
+authors:
+  - name: Paul Zierep*
+	orcid:
+	affiliation: 1
+  - name: Bérénice Batut
+	orcid: 0000-0001-9852-1987
+	affiliation: 1, 2
+  - name: Nicola Soranzo
+	orcid: 0000-0003-3627-5340
+	affiliation: 3
+  - name: Matúš Kalaš
+	orcid:
+	affiliation: 4
+  - name: Ove Johan Ragnar Gustafsson
+	orcid: 0000-0002-2977-5032
+	affiliation: 5
+affiliations:
+  - name: Bioinformatics Group, Department of Computer Science, University of Freiburg, Freiburg, Germany
+	index: 1
+  - name: Institut français de Bioinformatique, CNRS UAR 3601, Évry, France & Mésocentre Clermont-Auvergne, Université Clermont Auvergne, Aubiere, France
+	index: 2
+  - name: Earlham Institute, Norwich Research Park, Norwich, UK
+	index: 3
+  - name: 
+	index: 4
+  - name: Australian BioCommons, University of Melbourne, Melbourne, Victoria, Australia
+	index: 5
+date: 12 February 2024
+bibliography: paper.bib
+event: BioHackathon Europe 2023
+biohackathon_name: "BioHackathon Europe 2023"
+biohackathon_url:   "https://biohackathon-europe.org/"
+biohackathon_location: "Barcelona, Spain, 2023"
+group: BioHackrXiv
+git_url: https://github.com/galaxyproject/galaxy_tool_metadata_extractor/tree/e604ffd9866f9c32f797ceee45062b4d6f2a1513/docs/publications/BioHackrXiv
+authors_short: Zierep, P., Batut, B. \emph{et al.} (2024) BioHackEU23 project #25: Finding Galaxy tools
+---
+
+
+# Introduction or Background
+
+Galaxy[@10.1093/nar/gkac247] is a web-based analysis platform offering almost 10,000 different tools, which are developed across various GitHub repositories. Furthermore, Galaxy also embraces granular implementation of software tools as sub-modules. In practice, this means that tool suites are separated into Galaxy tools, also known as wrappers, that capture their component operations. Some key examples of suites include [Mothur](https://bio.tools/mothur)[@doi:10.1128/AEM.01541-09] and [OpenMS](https://bio.tools/openms)[@rost2016openms], which translate to tens and even hundreds of Galaxy tools. While granularity supports the composability of tools into rich domain-specific workflows, this decentralized development and sub-module architecture makes it difficult for Galaxy users to find and reuse tools. It may also result in Galaxy tool developers duplicating efforts by simultaneously wrapping the same software. This is further complicated by a lack of tool metadata, which prevents filtering for all tools in a specific research community or domain, and makes it all but impossible to employ advanced filtering with ontology terms and operations like EDAM[@black2021edam]. The final challenge is also an opportunity: the global nature of Galaxy means that it is a big community. Solving the visibility of tools across this ecosystem and the potential benefits are far-reaching for global collaboration on tool and workflow development.
```

Comment:
```suggestion
Galaxy[@citesAsAuthority:10.1093/nar/gkac247] is a web-based analysis platform offering almost 10,000 different tools, which are developed in various GitHub repositories. Furthermore, Galaxy also embraces granular implementation of software tools as sub-modules. In practice, this means that tool suites are separated into sets of Galaxy tools, also known as Galaxy wrappers, that contain functionality of a corresponding sub-component. Some key examples of tool suites include [Mothur](https://bio.tools/mothur)[@citesAsAuthority:doi:10.1128/AEM.01541-09] and [OpenMS](https://bio.tools/openms)[@citesAsAuthority:rost2016openms], which translate to tens and even hundreds of Galaxy tools. While granularity supports the composability of tools into diverse purpose-specific workflows, this decentralised development and modular architecture make it difficult for Galaxy users to find and use tools. It may also result in Galaxy tool-wrapper developers duplicating efforts by simultaneously wrapping the same software. This is further complicated by the lack of tool metadata, which prevents filtering of tools as relevant for a specific scientific community or domain, and makes it impossible to employ advanced filtering by ontology terms like ones from EDAM[@citesAsAuthority:black2021edam]. The final challenge is also an opportunity: the global nature of Galaxy means that it is a big community. Solving the visibility of tools across this "ecosystem", and the potential benefits are far-reaching for global collaboration on development of tools and workflows.
```

Sad the GitHub can only show a small maximum numbers of changes per line, making it unusable for editing articles in markdown 😕 In summary, I just added specifiers to the citations, and at some places changed our "Galaxish" into a more end-user English.